eng/pipelines/common/templates/jobs/validate-signed-package-job.yml

```diff
@@ -208,7 +237,7 @@ jobs:
         }
       }
     displayName: 'Verify all DLLs unzipped match "expected" hierarchy'
-
+  
```

Comment:
```suggestion

```

Reply:
there is an extra line in after each task, why should we remove this one?